fe/src/main/java/org/apache/doris/planner/StreamLoadPlanner.java

```diff
@@ -185,6 +185,10 @@ public TExecPlanFragmentParams plan(TUniqueId loadId) throws UserException {
                 }
                 partitionIds.add(part.getId());
             }
+        } else {
+            for (Partition partition : destTable.getPartitions()) {
```

Comment:
Add a comment to explain why we return all partitions.

Reply:
OK